frontend/src/App.js

```diff
@@ -1,15 +1,16 @@
 import React, { Component } from 'react';
```

Comment:
I'm reading that it's recommended to use .jsx extensions vs .js to help differentiate React views and components from pure javascript service files: https://reactjs.org/docs/introducing-jsx.html

Let's rename these extensions to .jsx to follow the docs